identity/handler_test.go

```diff
@@ -566,6 +567,33 @@ func TestHandler(t *testing.T) {
 		})
 	})
 
+	t.Run("suite=PATCH identities", func(t *testing.T) {
+		req := &identity.BatchPatchIdentitiesBody{
+			Identities: []*identity.BatchIdentityPatch{
+				{
+					Create: &identity.CreateIdentityBody{Traits: []byte(`{"email": "batch-import-1@ory.sh"}`)},
+				},
+				{
+					Create: &identity.CreateIdentityBody{Traits: []byte(`{"email": "batch-import-2@ory.sh"}`)},
+				},
+				{
+					Create: &identity.CreateIdentityBody{Traits: []byte(`{"email": "batch-import-3@ory.sh"}`)},
+				},
+				{
+					Create: &identity.CreateIdentityBody{Traits: []byte(`{"email": "batch-import-4@ory.sh"}`)},
+				},
+			},
+		}
+		res := send(t, adminTS, "PATCH", "/identities", http.StatusOK, req)
+		assert.Len(t, res.Get("identities").Array(), len(req.Identities))
+		assert.Empty(t, res.Get("identities.0.error").Raw)
+		identityID := res.Get("identities.0.identity").String()
+		require.NotEmpty(t, identityID)
+
+		getIdentity := get(t, adminTS, "/identities/"+identityID, http.StatusOK)
+		assert.Equal(t, "batch-import-1@ory.sh", getIdentity.Get("traits.email").String())
```

Comment:
I would prefer a snapshotx assertion here to ensure all fields are correctly imported.

Reply:
Snapshots only work for some fields, as the order of multiple (verifiable & recovery) addresses is random.